### PR TITLE
fixes #35 and code cleanup

### LINKED ISSUE #35: Update SolidCropPathPainter make paint style config able like DottedCropPathPainter

### PR DESCRIPTION
**Pull request by sikandernoori**
fixes #35 
@ikbendewilliam code cleanup and small improvement in SolidCropPathPainter

### CODE REVIEW

File: lib/src/painters/solid_path_painter.dart [hunk 1]
```diff
@@ -4,21 +4,33 @@ import 'package:flutter/material.dart';
...
-  final _paint = Paint()
-    ..color = Colors.white
-    ..strokeWidth = _strokeWidth
-    ..style = PaintingStyle.stroke
-    ..strokeJoin = StrokeJoin.round;
+  final Paint pathPaint;
 
   /// Draw a solid path around the given path
-  SolidCropPathPainter(this._path);
+  SolidCropPathPainter(this._path, {required this.pathPaint});
```
Comment (ikbendewilliam) [inline lib/src/painters/solid_path_painter.dart:10]: best to keep them the same, both named or both positioned. My preference goes to names, but either is fine. Just not one positioned and one named

**Review by ikbendewilliam — CHANGES_REQUESTED** [review]

File: lib/src/painters/solid_path_painter.dart (hunk 1 above)
Comment (sikandernoori) [reply lib/src/painters/solid_path_painter.dart]: Done.

**Review by ikbendewilliam — APPROVED** [review]

**Comment by sikandernoori** [comment]
@ikbendewilliam can we merge and push to pub.dev.

**Comment by ikbendewilliam** [comment]
Sure! Give me a second

**Comment by ikbendewilliam** [comment]
New version publishing, I've added the radius also to the example to showcase it.

**Pull request merged**